src/main/webapp/resources/js/jquery-ui.js

```diff
@@ -0,0 +1,18706 @@
+/*! jQuery UI - v1.12.1 - 2016-09-14
```

Comment:
Do we want to include jQuery UI in our source tree? Should we get it from a CDN instead? I'm not sure what we usually do.

Reply:
removed